iotc-azrtos-sdk/src/iotconnect.c

```diff
@@ -296,36 +301,39 @@ UINT iotconnect_sdk_init(IotConnectAzrtosConfig *ac) {
 	memcpy(&azrtos_config, ac, sizeof(azrtos_config));
     memset(&iic, 0, sizeof(iic));
 
+    last_sync_result = IOTCL_SR_UNKNOWN_DEVICE_STATUS;
 #ifndef PROTOCOL_V2_PROTOTYPE
 
-    // TODO: ALLOW CACHING! ---------------------------------------------------------------------------------
+    // TODO: ALLOW CACHING! --------------------------------------------------------------------------------
 	iotcl_discovery_free_discovery_response(discovery_response);
 	iotcl_discovery_free_sync_response(sync_response);
 	discovery_response = NULL;
 	sync_response = NULL;
     //if (!discovery_response) {
+		printf("IOTC: Performing discovery.\r\n");
         discovery_response = run_http_discovery(config.cpid, config.env);
         if (NULL == discovery_response) {
             // get_base_url will print the error
             return -1;
         }
-        printf("Discovery response parsing successful.\r\n");
+        printf("IOTC: Discovery response parsing successful.\r\n");
     //}
 
     //if (!sync_response) {
```

Comment:
Meant to be commented out? Maybe remove?

Reply:
Well the commented out code would be used if we decide to allow SYNC caching. It's probably safest to never cache sync. We will remove the commented code.

Reply:
fixed.